service/history/handler.go

```diff
@@ -1564,10 +1564,13 @@ func (h *Handler) ReapplyEvents(
 	if err != nil {
 		return h.error(err, scope, domainID, workflowID)
 	}
+
+	executionInfo := request.GetRequest().GetWorkflowExecution()
```

Comment:
executionInfo is usually used by mutable state, i guess this var is a typo and should be called execution